translations/translations.json

```
@@ -4427,6 +4427,18 @@
       "disclaimer": "版本说明"
     }
   },
+  "licenses": {
+    "de": {
+      "version": "Version:",
+      "license": "Lizenz:",
+      "noLicensesMessage": "Keinen Lizenzinformationen verfügbar"
+    },
+    "en": {
+      "version": "version:",
+      "license": "license:",
```

Since its the first word I'd use uppercase I think
```suggestion
      "version": "Version:",
      "license": "License:",
```